Source/iOS/OIDExternalUserAgentIOS.m

```diff
@@ -95,15 +95,15 @@ - (BOOL)presentExternalUserAgentRequest:(id<OIDExternalUserAgentRequest>)request
                                    callbackURLScheme:redirectScheme
                                    completionHandler:^(NSURL * _Nullable callbackURL,
                                                        NSError * _Nullable error) {
-      _authenticationVC = nil;
+      self->_authenticationVC = nil;
```

Comment:
I think we should only give this block a weak reference to self, since this block is owned by SFAuthenticationSession.

The other uses you migrated look fine because those blocks are all deterministically executed and released (by dispatch_async) - but I don't think we should be making any assumptions about what an SFAuthenticationSession does with the completion block it owns.

It looks like maybe setting _authenticationVC to nil in `cleanup` was breaking the circular reference before - I can't be sure without looking harder.

Reply:
Correct! I've changed it to a weak reference.